src/Agent.Worker/Handlers/HandlerFactory.cs

```diff
@@ -55,6 +55,10 @@ public sealed class HandlerFactory : AgentService, IHandlerFactory
             IHandler handler;
             if (data is BaseNodeHandlerData)
             {
+                // Node 6
+                if (data is NodeHandlerData) {       
+                    executionContext.Warning("This task uses Node 6 execution handler, which will be deprecated soon. If you are the owner of the task - please follow the migration to Node 10 handler. If you are the user - feel free to reach out to the owners of this task to proceed on migration.");
```

Comment:
Does it make sense to use the loc string here? 

> If you are the owner of the task - please follow the migration to Node 10 handler.

I believe it makes sense to add a link to the migration guide.